config.json

```diff
@@ -642,9 +642,12 @@
       "slug": "diffie-hellman",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Cryptography",
+        "Integers",
+        "Algorithms",
+        "Transforming"
```

Comment:
Yeah, we should avoid any confusion regarding Go interface.  May we choose a topic which isn't in the TOPICS.txt ?  If so, then Standard library sounds good.